docs/user-guide/installation.md

```diff
@@ -6,6 +6,9 @@ Polars is a library and installation is as simple as invoking the package manage
 
     ``` bash
     pip install polars
+
+    # Or for legacy CPUs without AVX2 support
+    pip install polars-ltc-cpu
```

Comment:
```suggestion
    pip install polars-lts-cpu
```

Reply:
@stinodego no problem. Do you think the section should be located elsewhere, or as a new section in the same installation page?

Reply:
This comment was not resolved, and the typo is now in the user guide 😕